package.json

```diff
@@ -3,7 +3,7 @@
   "scripts": {
     "build": "rm -rf ./dist/ && rollup --config && cp app.json dist/ && node build_bundle.js dist/",
     "bundle": "node build_bundle.js dist",
-    "test": "node  --trace-warnings --experimental-vm-modules node_modules/jest/bin/jest.js",
```

Comment:
this is because of the --experimental-vm-modules. It produces output which looks like errors. I did not yet find a final solution for these warnings. I believe we need to wait for future releases of jest.